### PR TITLE
Show upcoming events on home page

### PR DESCRIPTION
**Pull request by alsahawneh**

### CODE REVIEW

**Comment by thread-koder** [comment]
Hello @alsahawneh, hope you are having a nice day, there is something wrong with the CSS of the upcoming events section as you can see in the image that I attached
image

File: pages/index.vue [hunk 1]
```diff
@@ -2,6 +2,14 @@
   <div>
     <!-- Banner -->
     <homeBanner :page-meta="homeMeta" :programs="programs" />
+    <!-- Upcoming Events -->
+    <div class=container mt-20>
+      <eventList
```
Comment (thread-koder) [inline pages/index.vue:7]: @alsahawneh in the eventList component you marked the category prop as required in the component yet you didn't pass it here, if you allow me I have some suggestions, you could change the category prop from required to giving it a default value and required to false or pass the category to the component so we can get rid of the error.
Thank you.

Comment (alsahawneh) [reply pages/index.vue:7]: Thanks, you were right. I have just noticed that the event section is not vertically aligned with the heading above. I am currently working on that. Will push the changes ASAP.

Comment (alsahawneh) [reply pages/index.vue:7]: Apart from that, @mahasneh do you think it's better to create a new component for EventsSpotlight or to keep the code as it is?

**Review by mahasneh — CHANGES_REQUESTED** [review]

**Review by mahasneh — CHANGES_REQUESTED** [review]
Change the code as commented.

File: components/Events/EventsSpotlight.vue [hunk 2]
```diff
@@ -0,0 +1,163 @@
+<!-- Show events only if there are upcoming events -->
+<template>
+  <section>
```
[the remark this answers is not in the document]
Comment (alsahawneh) [reply components/Events/EventsSpotlight.vue]: Done.

**Pull request merged**